client/js/utils.js

```diff
@@ -37,6 +39,15 @@ function resetHeight(element) {
 	element.style.height = element.style.minHeight;
 }
 
+// Given a channel element will determine if the lounge user is Op in that channel
+function isOpInChannel(channel) {
```

Comment:
I wonder if this should be `function isOpInChannel(user, channel)`?

Reply:
Doesn't make sense in the context of this commit as I specifically moved all of that (including the nick detection) to that function and would mean that I would have to move the first three lines back to [here](https://github.com/creesch/lounge/blob/46dfad0a0045ceede19718cc4859575a9675b7e6/client/js/lounge.js#L101)  where I explicitly removed it as to keep that area clean and readable.